checks/python_module_version.go

```diff
@@ -0,0 +1,68 @@
+package checks
+
+import (
+	"fmt"
+	"os/exec"
+	"strings"
+)
+
+func init() {
+	availableChecks["python-module-version"] = func(args map[string]interface{}) (Checker, error) {
+		return PythonModuleVersion{}.FromArgs(args)
+	}
+}
+
+// PythonModuleVersion checks if python module is installed on the system
+// And verifies its version.
+//
+// Type:
+//   - python-module-version
+//
+// Support Platforms:
+//   - Linux
+//   - Windows
+//
+// Argument:
+//   name (required): A string value that represents the gem name
+//   version (optional): An optional version number to check
+// Leave version blank, if you just want to verify that a module is present
+type PythonModuleVersion struct {
+	Name    string
+	Version string
+}
+
+// Check if a python module is installed on the system and verify version if
+// the Version string is set
+func (pmv PythonModuleVersion) Check() error {
+	pyCommand := "import " + pmv.Name + "; print(" + pmv.Name + ".__version__)"
+	out, err := exec.Command("python", "-c", pyCommand).Output()
```

Comment:
We should use CombinedOutput and return the string(out) in the fmt.Errorf return to help debuggability. See [script.go](https://github.com/chasinglogic/redalert/blob/master/checks/script.go#L87) for an example. (Though the err.Error() call there is extraneous)